zbus/src/connection/mod.rs

```diff
@@ -844,8 +844,8 @@ impl Connection {
     /// # // https://gitlab.freedesktop.org/zeenix/zbus/-/jobs/34023494
     /// # #[cfg(all(not(feature = "tokio"), not(target_os = "windows")))]
     /// # {
+    /// use async_global_executor::{block_on, spawn};
```

Comment:
This comment needs to be adjusted

```
    /// Here is how one would typically run the zbus executor through async-std's single-threaded
    /// scheduler:
```

We could also use `smol` instead of `async-global-executor`. But that might pull more dependencies.